build.gradle

```diff
@@ -5,6 +5,9 @@ plugins {
     id "jaci.openrio.gradle.GradleRIO" version "2018.01.22"
 }
 
+sourceCompatibility JavaVersion.VERSION_1_8
+targetCompatibility JavaVersion.VERSION_1_8
+
```

Comment:
Good catch.  It is required that we build with this version, so better to explicitly state it.